res/TensorFlowLiteRecipes/Net_Duplicate_Weights_000/test.rule

```diff
@@ -0,0 +1,5 @@
+# To check if RemoveDuplicateConstPass removes all duplicate consts
+
+RULE "VERIFY_FILE_FORMAT" $(verify_file_format) '=' 1
+
+RULE "CIRCLE_CONST_COUNT" $(op_count CONST) '=' 2
```

Comment:
@seanshpark,

I have added rule for circle2circle dredd test, but  i don't know what rule I should use here, could you please help, how I can count CONST node, because `RULE "CIRCLE_CONST_COUNT" $(op_count CONST) '=' 2` doesn't work, due to CONST is not an operation, as I understand

Reply:
dredd test uses `circle-inspect` tool to print operators but as CircleConst is not an operator, it doesn't show it.
I'm not sure which may be the best solution but I can think is to add an option to `circle-inspect` to print `CONST` for each `CircleCons`.
We need to revise
- `circle-inspect` to add option something like `--constants`
- `dredd-rule-lib/rule-lib.sh` to add something like `const_count`

If you like, you can change them or wait for someone to do it.
I'll post an issue for this.

Reply:
> dredd test uses `circle-inspect` tool to print operators but as CircleConst is not an operator, it doesn't show it. I'm not sure which may be the best solution but I can think is to add an option to `circle-inspect` to print `CONST` for each `CircleCons`. We need to revise
> 
>     * `circle-inspect` to add option something like `--constants`
> 
>     * `dredd-rule-lib/rule-lib.sh` to add something like `const_count`

Done